docker/vue-app/src/components/form_editor_view/CategoryItem.js

```diff
@@ -0,0 +1,138 @@
+        },
+        catID() {
+            return this.categoriesRecord.categoryID;
+        },
+        stapledForms() {
+            let stapledForms = [];
+            this.categoriesRecord.stapledFormIDs.forEach(id => stapledForms.push({...this.categories[id]}));
+            stapledForms = stapledForms.sort((eleA, eleB) => eleA.sort - eleB.sort );
+            return stapledForms;
+        },
+        isStapledToOtherForm() {
+            return this.allStapledFormCatIDs.includes(this.categoriesRecord.categoryID);
+        },
+        /**
+         * NOTE: uses LEAF XSSHelpers.js
+         * @returns {string} truncated category name for card title
+         */
+        categoryName() {
+            return this.categoriesRecord.categoryName === '' ? 
+                'Untitled' : this.stripAndDecodeHTML(this.categoriesRecord.categoryName);
+        },
+        formDescription() {
+            return this.stripAndDecodeHTML(this.categoriesRecord.categoryDescription);
+        },
+        /**
+         * 
+         * @returns {string} workflow ID and description
+         */
+        workflowDescription() {
+            let msg = '';
+            if (this.workflowID > 0) {
+                msg = `${this.categoriesRecord.workflowDescription || 'No Description'} (#${this.categoriesRecord.workflowID})`;
+            } else {
+                msg = 'No Workflow';
+            }
+            return msg;
+        }
+    },
+    methods: {
+        updateSort(event = {}, categoryID = '') {
+            let sortValue = parseInt(event.currentTarget.value);
+            if(isNaN(sortValue)) return;
+
+            if (sortValue < -128) {
+                sortValue = -128;
+                event.currentTarget.value = -128;
+            }
+            if (sortValue > 127) {
+                sortValue = 127;
+                event.currentTarget.value = 127;
+            }
+            $.ajax({
+                type: 'POST',
+                url: `${this.APIroot}formEditor/formSort`,
+                data: {
+                    sort: sortValue,
+                    categoryID: categoryID,
+                    CSRFToken: this.CSRFToken
+                },
+                success: () => {
+                    this.updateCategoriesProperty(categoryID, 'sort', sortValue);
+                },
+                error: err => console.log('sort post err', err)
+            })
+        }
+    },
+    template:`<tr :id="catID" :title="catID + ': ' + categoryName">
+            <td height="40" class="form-name">
+                <router-link :to="{ name: 'category', query: { formID: catID }}">
+                {{ categoryName }}
+                </router-link>
+            </td>
+            <td class="formPreviewDescription">{{ formDescription }}</td>
+            <td v-if="availability !== 'supplemental'">{{ workflowDescription }}</td>
+            <td v-if="availability === 'supplemental'">

```

Comment:
v-else?

Reply:
Added the alternative column later and didn't catch this - updated.